model/deliveryLog/event/DeliveryLogEvent.php

```diff
@@ -37,13 +37,14 @@ class DeliveryLogEvent implements Event
     public const EVENT_ID_TEST_AUTHORISE = 'TEST_AUTHORISE';
     public const EVENT_ID_TEST_TERMINATE = 'TEST_TERMINATE';
     public const EVENT_ID_TEST_IRREGULARITY = 'TEST_IRREGULARITY';
+    public const EVENT_ID_TEST_ADJUSTED_TIME = 'TEST_ADJUSTED_TIME';
```

Comment:
Probably would be better to use the same tense for all events.

Reply:
fixed